go/pkg/client/tree_test.go

```diff
@@ -692,28 +697,27 @@ func TestComputeMerkleTree(t *testing.T) {
 			input: []*inputPath{
 				{path: "foobarDir/foo", fileContents: fooBlob, isExecutable: true},
 				{path: "foobarDir/bar", fileContents: barBlob},
-				{path: "foobarSymDir", isSymlink: true, isAbsolute: true, symlinkTarget: "foobarDir"},
+				{path: "base/foobarSymDir", isSymlink: true, isAbsolute: true, symlinkTarget: "foobarDir"},
```

Comment:
This might seem like it should be `../foobarDir`, but because `isAbsolute` is `true`, the actual target it generates is `"${execRoot}/foobarDir"`.